src/splitAvgMinDistance.test.js

```diff
@@ -0,0 +1,22 @@
+const {splitAvgMinDistance} = require("./splitAvgMinDistance.js");
+
+test("Return values of splitAvgMinDistance", () => {
+    let cTwoSix = [
+        [9 / 11, 7 / 11, 9 / 11, 3 / 11],
+        [5 / 11, -3 / 11, 1 / 11, -5 / 11],
+        [1 / 11, 5 / 11, 5 / 11, -3 / 11],
+        [-5 / 11, -7 / 11, 3 / 11, -9 / 11],
+        [7 / 11, -7 / 11, 11 / 11, -9 / 11],
+        [1 / 11, 5 / 11, 5 / 11, -7 / 11],
+        [9 / 11, 5 / 11, 5 / 11, -3 / 11]];
```

Comment:
Lad den afsluttende ] være på en linje for sig selv. Det skal den øverste [ også egentlig

Reply:
Så er det gjort